pkg/cli/common.go

```diff
@@ -41,12 +47,31 @@ func getUniqueFileNode(fnode *service.FileNode) (*service.FileNode, error) {
 }
 
 // store nodes in a File node array
-func createFileNodesArray(these []interface{}) []*service.FileNode {
+func createFileNodesArray(these []interface{}) ([]*service.FileNode, error) {
 	var theseFileNodes []*service.FileNode
 	for _, fNode := range these {
-		theseFileNodes = append(theseFileNodes, fNode.(*service.FileNode))
+		switch this := fNode.(type) {
```

Comment:
Are there more cases about to be added here. Otherwise switch with one case is not worth a switch.